br/pkg/lightning/config/config.go

```diff
@@ -98,7 +98,7 @@ const (
 var (
 	supportedStorageTypes = []string{"file", "local", "s3", "noop", "gcs", "gs"}
 
-	DefaultFilter = []string{
+	defaultFilter = []string{
```

Comment:
How about change this variable from `var` to `const`

Reply:
Golang doesn't support constant string slice.  Compile error:  `[]string{…} (value of type []string) is not constant`